hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -101,12 +101,10 @@ def defines_shrink_pass(generate_arguments):
     """A convenient decorator for defining shrink passes."""
 
     def accept(run_step):
-        definition = ShrinkPassDefinition(
-            generate_arguments=generate_arguments, run_step=run_step
-        )
+        ShrinkPassDefinition(generate_arguments=generate_arguments, run_step=run_step)
 
         def run(self):
-            return self.run_shrink_pass(definition.name)
+            assert False, "Shrink passes should not be run directly"
```

Comment:
This is what's causing the coverage failure, so presumably it just needs to be suppressed.

Reply:
Huh. I thought the `assert False` was automatically a coverage exclusion. Oh well, easy enough to add the pragma.